dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Components/GeneratorWithControl/Controls/ExciterAvrPssOel.mo

```diff
@@ -0,0 +1,155 @@
+within Dynawo.Examples.Nordic.Components.GeneratorWithControl.Controls;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model ExciterAvrPssOel
+
+  import Modelica;
+  import Dynawo.Connectors;
+
+  parameter Real G;
```

Comment:
Could you please :
- Add a comment for each parameter to describe it. The comment should give the unit (in kA or in pu) and the pu base if needed (in pu (base UNom, SnRef) for example)
- Use the Dynawo Types instead of Real when possible. For example parameter Types.CurrentModulePu IfLimPu - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Reply:
resolved, see [e1000c7](https://github.com/dynawo/dynawo/pull/2394/commits/e1000c7adff25e271a2e2214bb7197d005b00633)